client/src/components/details/productInformation.jsx

```diff
@@ -10,6 +10,24 @@ import AddToBag from './addToBag.jsx';
 import Favorite from './favorite.jsx';
 
 const ProductInformation = ({product, styleList, favorite, setFavorite, currentStyle, setCurrentStyle}) => {
+  // functions for conditional rendering: review/rating, price
+  const reviewsAndRatings = () => {
+    const hasReviews = product.reviewsMeta.reviewsCount > 0;
+    return (
+      hasReviews
+      ?
+      <DetailsLib.rating>
+      <StyleLib.small> Average Rating: </StyleLib.small> <Stars rating={product.reviewsMeta.averageRating} />
+        <DetailsLib.smallUnderline>{'Read All ' + product.reviewsMeta.reviewsCount + ' Reviews'}</DetailsLib.smallUnderline>
+      </DetailsLib.rating>
+      :
+      <DetailsLib.rating>
+      <StyleLib.small></StyleLib.small>
+        <DetailsLib.smallUnderline></DetailsLib.smallUnderline>
+      </DetailsLib.rating>
+    );
+  }
+
```

Comment:
nitpick - It looks like you're creating a reviewAndRatings sub-component inside the productInformation component and then returning it below. It feels like bad practice to create sub-components in their parent component. I suggest separating this into it's own file and then importing it in to this file as a component so you have two smaller files with clearer separations of concerns.